templates/skeleton/package.json

```diff
@@ -14,10 +14,11 @@
   "prettier": "@shopify/prettier-config",
   "dependencies": {
     "@remix-run/react": "2.1.0",
+    "@remix-run/server-runtime": "2.1.0",
     "@shopify/cli": "3.50.0",
     "@shopify/cli-hydrogen": "^6.0.0",
-    "@shopify/hydrogen": "^2023.10.0",
-    "@shopify/remix-oxygen": "^2.0.0",
+    "@shopify/hydrogen": "~2023.10.2",
```

Comment:
Also, using `^` in our templates for Hydrogen leads to getting new major versions. Better to use `~` for this one, or even pin to our latest release every time. Thoughts?

Reply:
We just need to make sure a resolved project doesn't have duplicate remix dependencies.